tests/Domain/data/notification_poll_invalid_count.php

```diff
@@ -0,0 +1,6 @@
+<?php declare(strict_types = 1);
+
+return [
+    'count' => 1,
+    'notification' => [''],
```

Comment:
As I have said before, the value of `notification` should be a notification object, you could include the `notification_valid.php` here for convenience.